### PR TITLE
New hybrid template with server side project (for Lightning Web Components)

### PR DESCRIPTION
**Pull request by wmathurin**
Corresponding PR in the package repo: https://github.com/forcedotcom/SalesforceMobileSDK-Package/pull/203

Screenshot 2020-06-18 18 00 01

### CODE REVIEW

File: HybridLwcTemplate/server/force-app/main/default/aura/LightningDependencyApp/LightningDependencyApp.app-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AuraDefinitionBundle xmlns="urn:metadata.tooling.soap.sforce.com" fqn="LightningDependencyApp">
+    <apiVersion>46.0</apiVersion>
```
Comment (bhariharan) [inline HybridLwcTemplate/server/force-app/main/default/aura/LightningDependencyApp/LightningDependencyApp.app-meta.xml:3]: Could this be auto-pulled from the SDK's `apiVersion`?

File: HybridLwcTemplate/server/force-app/main/default/classes/ContactListController.cls-meta.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="urn:metadata.tooling.soap.sforce.com" fqn="ContactListController">
+    <apiVersion>46.0</apiVersion>
```
Comment (bhariharan) [inline HybridLwcTemplate/server/force-app/main/default/classes/ContactListController.cls-meta.xml:3]: Same question here - would be nice to line these up from a single place.

File: HybridLwcTemplate/server/force-app/main/default/lwc/contactList/contactList.js-meta.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="ContactList">
+    <apiVersion>46.0</apiVersion>
```
Comment (bhariharan) [inline HybridLwcTemplate/server/force-app/main/default/lwc/contactList/contactList.js-meta.xml:3]: One more.

File: HybridLwcTemplate/server/force-app/main/default/lwc/contactListItem/contactListItem.js-meta.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="ContactListItem">
+    <apiVersion>46.0</apiVersion>
```
Comment (bhariharan) [inline HybridLwcTemplate/server/force-app/main/default/lwc/contactListItem/contactListItem.js-meta.xml:3]: And another.

File: HybridLwcTemplate/template.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2016-present, salesforce.com, inc.
```
Comment (bhariharan) [inline HybridLwcTemplate/template.js:2]: `2020-present`.

File: HybridLwcTemplate/server/force-app/main/default/aura/LightningDependencyApp/LightningDependencyApp.app-meta.xml (hunk 1 above)
Comment (wmathurin) [reply HybridLwcTemplate/server/force-app/main/default/aura/LightningDependencyApp/LightningDependencyApp.app-meta.xml:3]: We have never done that for our js assets (e.g. https://github.com/forcedotcom/SalesforceMobileSDK-Shared/blob/dev/libs/force.js).
We typically just move up manually once in a while after verifying that there is no behavior change in the new version.

**Pull request merged**